src/staff-handbook/leave/TOIL.md

```diff
@@ -28,7 +28,9 @@ to take the time back.
 
 ## Requesting TOIL
 
-Using BreatheHR similar to the way that you request holiday, there is a TOIL
-drop down, please write a note as to why you are requesting the leave and this
-will be approved by a member of the People team. Please note this will not deduct
-from your holiday allowance shown on your BreatheHR dashboard.
+Using BreatheHR, go to **My dashboard** and click the link **Log TOIL**,
+fill in the form and click the **log TOIL** button.
+Please write a note as to why you are requesting the TOIL and this
+will be approved by your line manager.
+Once approved, your TOIL will be added to your holiday allowance.
+You should then book leave as normal to have time not working.
```

Comment:
Is there a reason for the line breaks? It condenses into one paragraph without double line breaks, but is it to divide each point up in the code?

Reply:
Not really. In my last place I followed a rule of "one sentence per line" which makes code review much easier, but I'll tidy this up before merging.